packages/console/src/pages/UserDetails/UserSettings/UserConnections/UserSocialIdentities.tsx

```diff
@@ -0,0 +1,165 @@
+import type {
+  ConnectorResponse,
+  GetUserAllIdentitiesResponse,
+  DesensitizedSocialTokenSetSecret,
+} from '@logto/schemas';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import useSWR from 'swr';
+
+import UnnamedTrans from '@/components/UnnamedTrans';
+import Button from '@/ds-components/Button';
+import CopyToClipboard from '@/ds-components/CopyToClipboard';
+import FormField from '@/ds-components/FormField';
+import ImageWithErrorFallback from '@/ds-components/ImageWithErrorFallback';
+import Table from '@/ds-components/Table';
+import type { RequestError } from '@/hooks/use-api';
+import { getConnectorGroups } from '@/pages/Connectors/utils';
+
+import ConnectorTokenStatus from './ConnectorTokenStatus';
+import styles from './index.module.scss';
+
+type Props = {
+  readonly identities?: GetUserAllIdentitiesResponse['socialIdentities'];
+  readonly isIdentitiesLoading?: boolean;
+};
+
+type RowData = {
+  target: ConnectorResponse['target'];
+  identityId?: string;
+  logo?: ConnectorResponse['logo'];
+  name: ConnectorResponse['name'] | string;
+  tokenStatus: {
+    isTokenStorageSupported?: boolean;
+    tokenSecret?: DesensitizedSocialTokenSetSecret;
+  };
+};
+
+function ConnectorName({ name }: { readonly name: RowData['name'] }) {
+  return typeof name === 'string' ? <span>{name}</span> : <UnnamedTrans resource={name} />;
+}
+
+function UserSocialIdentities({ isIdentitiesLoading, identities = [] }: Props) {
+  const { t } = useTranslation(undefined, {
+    keyPrefix: 'admin_console',
+  });
+
+  const {
+    data: connectors,
+    error,
+    isLoading,
+    mutate,
+  } = useSWR<ConnectorResponse[], RequestError>('api/connectors');
+
+  const isDataLoading = useMemo(
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+    () => isIdentitiesLoading || isLoading,
+    [isIdentitiesLoading, isLoading]
+  );
```

Comment:
Simple boolean value does not deserve a `useMemo`. Please only use `useMemo` for complex calculations.